### PR TITLE
feat: Contract Calls Chart 

### LINKED ISSUE #534: Contract Calls Chart
### Please describe the problem that your request should resolve.
Display contract calls chart

### Describe the solution you'd like
The chart displays the number of calls per time unit

### Is it already possible to achieve the same outcome in any other way? If so, how?
Endpoint https://mainnet.aeternity.io/mdw/v3/statistics/transactions?tx_type=contract_call&limit=1000

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #534 

## Demo

firefox_rGWjVfz5uH

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add contracts chart

### CODE REVIEW

File: src/components/ContractsChartPanel.vue [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      CONTRACT CALLS
+    </template>
+    <template #header>
+      <contracts-chart-controls
+        class="contracts-chart-panel__chart-controls"
+        @selected="loadContractsStatistics"/>
+    </template>
+
+    <div class="contracts-chart-panel__container">
+      <Line
```
Comment (janmichek) [inline src/components/ContractsChartPanel.vue:13]: With the next chart I will do even more abstraction to reuse code and eliminate code duplication.

File: src/components/ChartControls.vue [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="transactions-chart-controls">
+  <div class="chart-controls">
```
Comment (janmichek) [inline src/components/ChartControls.vue:2]: Partial abstraction

File: src/components/TransactionsChartPanel.vue [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <template>
   <app-panel>
     <template #heading>
-      TOTAL TRANSACTIONS
+      TRANSACTIONS TREND
```
Comment (janmichek) [inline src/components/TransactionsChartPanel.vue:4]: I decided to rename the headings, inspired by reference charts

**Review by michele-franchi — APPROVED** [review]
🚀

**Comment by janmichek** [comment]
@Liubov-crypto I also touched the code of Transaction Chart. It would be nice to check as well

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Pull request merged**